src/server/game/Battlegrounds/Zones/BattlegroundWS.cpp

```diff
@@ -381,6 +383,13 @@ void BattlegroundWS::EventPlayerCapturedFlag(Player* player)
         _flagsTimer[GetTeamIndexByTeamId(player->GetTeam()) ? 0 : 1] = BG_WS_FLAG_RESPAWN_TIME;
     }
 }
+void BattlegroundWS::HandleFlagRoomCapturePoint(int32 team)
+{
+    Player* flagCarrier = ObjectAccessor::GetPlayer(GetBgMap(), GetFlagPickerGUID(team));
+    uint32 areaTrigger = team == TEAM_ALLIANCE ? 3647 : 3646;
```

Comment:
Magic numbers? :)

Reply:
are the same here: https://github.com/TrinityCore/TrinityCore/blob/bbf4cd5ac6b12f76e719c9498868b706145be0db/src/server/game/Battlegrounds/Zones/BattlegroundWS.cpp#L674

(and same in the other bg scripts)

Reply:
You can be the first one who changes that :)

Reply:
Better in other pr, I prefer not to mix